gallery/src/components/NxDropdown/NxDropdownPage.tsx

```diff
@@ -132,7 +132,7 @@ const NxDropdownPage = () =>
               </td>
             </tr>
             <tr className="nx-table-row">
-              <td className="nx-cell">HTML div Attributes</td>
+              <td className="nx-cell">HTML <code className="nx-code">&lt;div&gt;</code> Attributes</td>
               <td className="nx-cell">
                 <a target="_blank"
                    rel="noopener"
```

Comment:
I've noticed that some of these, such as this one, link to the spanish docs for some reason

Reply:
> I've noticed that some of these, such as this one, link to the spanish docs for some reason

Is @raveneyex pranking us?

https://github.com/sonatype/sonatype-react-shared-components/pull/397/commits/82d24796b1adbde73e81ade471cf738536ac5909

Reply:
> Is @raveneyex pranking us?

The links are now broken.  You missed that the spanish links use "Elemento" instead of "Element"

Reply:
> > Is @raveneyex pranking us?
> 
> The links are now broken. You missed that the spanish links use "Elemento" instead of "Element"

Fixed (and checked this time): https://github.com/sonatype/sonatype-react-shared-components/pull/397/commits/b57f4f5a9b6f3d5d53aa755edf10d6831a270fc6